pom.xml

```diff
@@ -34,6 +34,14 @@
         </developer>
     </developers>
 
+    <distributionManagement>
+        <repository>
+            <id>bintray-algolia-maven</id>
+            <name>algolia-maven</name>
+            <url>https://api.bintray.com/maven/algolia/maven/algoliasearch/;publish=1</url>
+        </repository>
+    </distributionManagement>
```

Comment:
Let's do this in another PR: it is unrelated to switching to multi-modules, and it can break things badly, so let's keep the two separate (so if we need to revert, we can revert just the use of bintray)